_posts/2015-10-19-quickstart-install-kissmetrics.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: post
+categories: getting-started
+title: "Quickstart Guide: Install Kissmetrics’ JavaScript Tracking Code, Create Custom Events and Properties, and Identify Users"
+summary: "In this post, we'll be go through each step you need to take to: (1) get the JavaScript code on every page of your site, (2) understand what the Kissmetrics JavaScript snippet tracks, (3)  create custom events and properties, and (4) identify users of your product or service."
+---
+Before you can get data into your Kissmetrics account, you'll first need to install the JavaScript tracking code on every webpage of your site. This code is unique and specific to your account.
+
+In this post, we'll be go through each step you need to take to: (1) get the JavaScript code on every page of your site, (2) understand what the Kissmetrics JavaScript snippet tracks, (3)  create custom events and properties, and (4) identify users of your product or service.
+
+* Table of Contents
+{:toc}
+***
+
+##Four Ways to Edit Your Website’s Source Code
+There are plenty of ways to edit your website's source code:
+###Method 1 - Use Git
```

Comment:
Needs a blank new line between the last sentence and `###Method 1 ...`. Right now it's all being interpreted as one paragraph:

<img width="667" alt="screen shot 2015-10-20 at 12 13 12" src="https://cloud.githubusercontent.com/assets/1744567/10618300/06edba7e-7724-11e5-9bb8-8dec9ad31eb9.png">